### ISSUE TITLE
Issue with database migration

### ISSUE THREAD

**Issue by IanMayo** [debrief/pepys-import#453]
With our improved error logging, the client reported this:

==========

Alembic migration command running, see output below.
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade e2f70908043d -> 2a91be2822a1, Rename reference table cols to name
Exception details: (psycopg2.errors.UndefinedColumn) column "class_type" does not exist

[SQL: ALTER TABLE pepys."ClassificationTypes" RENAME class_type TO name]
(Background on this error at: http://sqlalche.me/e/f405)

ERROR: Alembic error when migrating the database!

==========

**Comment by IanMayo** [comment]
I just ran the current migration against a Postgres database, and it worked fine:
```
(pepys-admin) 5
Alembic migration command running, see output below.
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 30194fdfc033 -> d96db9a0620f, Add level attribute to Privacies Table
```

**Comment by BarisSari** [comment]
It seems that the client's database doesn't have `class_type` column in `ClassificationTypes` table. If this field is missing, probably the rest of the columns are missing, too: `level` column in `ConfidenceLevels` and `contact_type` in `ContactTypes`. Because the migration script with revision ID of _2a91be2822a1_ changes these fields. 

I downgraded and upgraded my Postgres DB without any problem. I tested upgrading-downgrading using both Alembic commands and Pepys Admin. Unfortunately, I couldn't produce the same error. 

Only one thing came up my mind which might be the cause of failure: Maybe the client already has a newer version of the database but the Alembic version is wrong in the database. If that's the case it's a bit difficult to solve because we should check the state of tables manually to decide which version of the database the client uses.
